repo_resource/common.py

```diff
@@ -105,6 +160,28 @@ def to_file(self, filename):
     def metadata(self) -> str:
         return ''
 
+    def standard(self) -> str:
+        try:
+            root = ET.fromstring(self.__version)
+            for element in root:
+                if element.tag not in TAGS:
+                    root.remove(element)
+                    print("Removed ", element.tag)
```

Comment:
Please don't print things. Printing things to stdout will break concourse checking mechanism